packages/grid_client/src/helpers/flists.ts

```diff
@@ -0,0 +1,159 @@
+export const flists = {
```

Comment:
I think we should make all constants uppercase https://javascript.info/variables#uppercase-constants
IMO should be FLIST.FULLVM_UBUNTU_20
As all of these values are hardcoded 

https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3990#issuecomment-2788786606

Reply:
@samaradel I think we need to do the same with flists 
```suggestion
export const FLISTS = {
```